src/pages/CheckUniqueIngredient.tsx

```diff
@@ -0,0 +1,5 @@
+const CheckUniqueIngredient = () => {
```

Comment:
```suggestion
const CheckUniqueIngredients = () => {
```

복수형으로 바꾸는것이 좋을 것 같습니다.

Reply:
유니크 재료가 하나라고 해서 단수형이 좋은 거 같은데 어떠세요?